src/game.js

```diff
@@ -405,9 +405,9 @@ var Game = Class.create({
 
 		}
 
-		this.activeCreature = this.players[0].creatures[0]; // Prevent errors
+		this.activeCreature = this.players[0].creatures[0]; // Prevents errors
 
-		this.UI = new UI(); // Creating UI not before because certain function requires creature to exists
+		this.UI = new UI(); // Creating UI (not before because some functions requires creatures to exist).
```

Comment:
No dot at the end, or we would need to use those everywhere.

Reply:
True, bad reflex of mine.